ts/Grid/Core/Options.ts

```diff
@@ -519,8 +519,9 @@ export interface CreditsOptions {
 
     /**
      * The text for the credits label.
+     * 
+     * Reference to Highcharts icon, that is enabled in Grid Lite, by default.
```

Comment:
The whole CreditsOptions interface should be removed from there and moved to `CreditsComposition` as a part of `declare module`.